aws-android-sdk-pinpoint/src/main/java/com/amazonaws/mobileconnectors/pinpoint/analytics/AnalyticsClient.java

```diff
@@ -359,33 +359,33 @@ public void removeGlobalMetric(String eventType, String metricName) {
     }
 
     /**
-     * Adds the specified campaign attributes to events to track Campaign Analytic
+     * Adds the specified pinpoint attributes to events to track Campaign/Journey Analytic
```

Comment:
Marketing nit: a capital `P` for `Pinpoint`, the proper-noun

(Apply same to all natural language portions of this body of work)

------------
Nit 2, it wasn't your bad, but the last person wrote `Analytic`. That's a word, I guess, but there are multiple analytics at play! Can we make it `analytics`?

Reply:
Yup...fixed both of those. I scanned the file for other occurrences as well.